### PR TITLE
FRPS Server Instructions

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds updated instructions for enabling an FRPS server. 

[VerteX Steps](https://deploy-preview-1474--docs-spectrocloud.netlify.app/vertex/system-management/reverse-proxy)

[Palette Steps](https://deploy-preview-1474--docs-spectrocloud.netlify.app/enterprise-version/reverse-proxy)

## Review Changes

💻 [Preview URL](https://deploy-preview-1474--docs-spectrocloud.netlify.app/vertex/system-management/reverse-proxy)

🎫 [PAC-719](https://spectrocloud.atlassian.net/browse/PAC-719)

### CODE REVIEW

**Review by tiwarisumit3 — APPROVED** [review]

File: content/docs/12-enterprise-version/07.1-reverse-proxy.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    {
+      "caCert": "-----BEGIN CERTIFICATE-----\n.............\n-----END CERTIFICATE-----",
+      "clientCert": "-----BEGIN CERTIFICATE-----\n..........\n-----END CERTIFICATE-----",
+      "clientKey": "-----BEGIN RSA PRIVATE KEY-----\n........\n-----END RSA PRIVATE KEY-----",
+      "port": 443,
+      "protocol": "https",
+      "server": "frps.palette.example.com.com"
+    }
+  ```
+
```
Comment (ritawatson) [inline content/docs/12-enterprise-version/07.1-reverse-proxy.md:159]: Let's add some space between the example payload and the infobox.

File: content/docs/12-enterprise-version/07.1-reverse-proxy.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+3. Using the output you received, copy the authorization value to your clipboard and assign it to a shell variable. Replace the authorization value below with the value from the output.
+
+  <br />
+
+  ```shell hideClipboard
+  TOKEN=**********
+  ```
+
+4. Query the system API endpoint `/v1/system/config/reverseproxy` to verify the current reverse proxy settings applied to Palette. Use the `curl` command below and replace the URL with the custom domain URL you assigned to Palette or use the IP address. You can use the `TOKEN` variable you created earlier for the authorization header.
```
Comment (ritawatson) [inline content/docs/12-enterprise-version/07.1-reverse-proxy.md:236]:
```suggestion
4. Query the system API endpoint `/v1/system/config/reverseproxy` to verify the current reverse proxy settings applied to Palette. Use the `curl` command below and replace the URL with the custom domain URL you assigned to Palette, or use the IP address. You can use the `TOKEN` variable you created earlier for the authorization header.
```

File: content/docs/12-enterprise-version/07.1-reverse-proxy.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+  <br />
+
+  ```bash
+  curl --location --request GET 'https://palette.example.com/v1/system/config/reverseproxy' \
+   --header "Authorization: $TOKEN"
+  ```
+
+  If the proxy server is configured correctly, you will receive an output similar to the following containing your settings. The SSL certificate outputs are truncated for brevity.
```
Comment (ritawatson) [inline content/docs/12-enterprise-version/07.1-reverse-proxy.md:245]:
```suggestion
  If the proxy server is configured correctly, you will receive an output similar to the following that contains your settings. The SSL certificate outputs are truncated for brevity.
```

File: content/docs/12.5-vertex/02-install-palette-vertex/20-install-on-kubernetes/10-install.md [hunk 4]
```diff
@@ -239,7 +240,15 @@ The following instructions are written agnostic to the Kubernetes distribution y
...
+10. After login, a summary page is displayed. Palette VerteX is installed with a self-signed SSL certificate. To assign a different SSL certificate you must upload the SSL certificate, SSL certificate key, and SSL certificate authority files to Palette VerteX. You can upload the files using the Palette VerteX system console. Refer to the [Configure HTTPS Encryption](/vertex/system-management/ssl-certificate-management) page for instructions on how to upload the SSL certificate files to Palette VerteX.
+
+<br />
+
+<WarningBox>
+
+If you are planning on deploying host clusters into different networks, you may require a reverse proxy, check out the [Configure Reverse Proxy](/vertex/system-management/reverse-proxy) guide for instructions on how to configure a reverse proxy for Palette VerteX.
```
Comment (ritawatson) [inline content/docs/12.5-vertex/02-install-palette-vertex/20-install-on-kubernetes/10-install.md:249]:
```suggestion
If you are planning to deploy host clusters into different networks, you may require a reverse proxy.  Check out the [Configure Reverse Proxy](/vertex/system-management/reverse-proxy) guide for instructions on configuring a reverse proxy for Palette VerteX.
```

File: content/docs/12.5-vertex/60-system-management/50-reverse-proxy.md [hunk 5]
```diff
@@ -30,7 +29,7 @@ You can deploy and maintain your own Spectro Proxy server. The Spectro Proxy ser
 - [Helm](https://helm.sh/docs/intro/install/) is installed and available.
 
 
-- Access to the target Kubernetes cluster's kubeconfig file. You must be able to interact with the cluster using `kubectl` commands and have sufficient permissions to install Palette VerteX. We recommend using a role with cluster-admin permissions to install Palette VerteX.
+- Access to the Palette VerteX Kubernetes cluster's kubeconfig file. You can download the kubeconfig file from the Palette VerteX system console. Navigate to **Enterprise System Migration**, select the Palette VerteX cluster, and click the **Download Kubeconfig** button for the cluster.
```
Comment (ritawatson) [inline content/docs/12.5-vertex/60-system-management/50-reverse-proxy.md:32]:
```suggestion
- Access to the kubeconfig file of the Palette VerteX Kubernetes cluster. You can download the kubeconfig file from the Palette VerteX system console. Navigate to **Enterprise System Migration**, select the Palette VerteX cluster, and click the **Download Kubeconfig** button for the cluster.
```

Comment (ritawatson) [reply content/docs/12.5-vertex/60-system-management/50-reverse-proxy.md:32]: FYI, the reason for this suggestion is that when appostrophe 's' is used with the plural in the same sentence, it's hard to read. You can hear it if you read it aloud.

**Review by ritawatson — APPROVED** [review]
Approved, but some wording suggestions need addressing.

File: content/docs/12-enterprise-version/07.1-reverse-proxy.md [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+
+- A domain name that you can use for the reverse proxy server. You will also need access to the DNS records for the domain so that you can create a CNAME DNS record for the reverse proxy server load balancer.
+
+
+- Ensure you have an SSL certificate that matches the domain name you will assign to Spectro Proxy. You will need this to enable HTTPS encryption for the Spectro Proxy. Contact your network administrator or security team to obtain the SSL certificate. You need the following files:
+  - x509 SSL certificate file in base64 format.
+
+  - x509 SSL certificate key file in base64 format.
+
+  - x509 SSL certificate authority file in base64 format.
```
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/07.1-reverse-proxy.md:43]:
```suggestion
  - x509 SSL certificate file in base64 format

  - x509 SSL certificate key file in base64 format

  - x509 SSL certificate authority file in base64 format
```

File: content/docs/12-enterprise-version/07.1-reverse-proxy.md [hunk 7]
```diff
@@ -0,0 +1,258 @@
...
+      "caCert": "-----BEGIN CERTIFICATE-----\n.............\n-----END CERTIFICATE-----",
+      "clientCert": "-----BEGIN CERTIFICATE-----\n..........\n-----END CERTIFICATE-----",
+      "clientKey": "-----BEGIN RSA PRIVATE KEY-----\n........\n-----END RSA PRIVATE KEY-----",
+      "port": 443,
+      "protocol": "https",
+      "server": "frps.palette.example.com.com"
+    }
+  ```
+
+  <InfoBox>
```
Comment (karl-cardenas-coding) [inline content/docs/12-enterprise-version/07.1-reverse-proxy.md:160]:
```suggestion
   <br />

  <InfoBox>
```

**Pull request merged**